grid-proxy/internal/gpuindexer/gpuindexer.go

```diff
@@ -150,7 +208,7 @@ func (n *NodeGPUIndexer) gpuNodeResultsBatcher(ctx context.Context) {
 	}
 }
 
-func (n *NodeGPUIndexer) Start(ctx context.Context) {
+func (n *NodeGPUIndexer) Start(ctx context.Context, connStr string) {
```

Comment:
I understand why you did it. however this isn't correct, this should be either inferred from n.db or defined as a method on the db struct

Reply:
went with defining method on the db that will return the connStr